src/main/java/com/puppycrawl/tools/checkstyle/checks/design/VisibilityModifierCheck.java

```diff
@@ -317,15 +545,36 @@
  * Example of allowed fields:
  * </p>
  * <pre>
- * class SomeClass
- * {
- *   &#64;CustomAnnotation
- *   String customAnnotated; // no warning
- *   &#64;com.annotation.CustomAnnotation
- *   String customAnnotated1; // no warning
- *   &#64;mypackage.annotation.CustomAnnotation
- *   String customAnnotatedAnotherPackage; // another package but short name matches
- *                                         // so no violation
+ * public final class MyClass {
+ * private int myPrivateField1;
+ *
+ * int myPrivateField3;               // violation, must have a visibility modifier
+ *
+ * protected String myProtectedField; // violation, protected visibility is not allowed
```

Comment:
please shorten names to bring trailing commend more to left to be readable ok on phone screens.

Reply:
done